test/integration/tcp.go

```diff
@@ -44,6 +44,10 @@ func (t *tcp) run() error {
 	funcs := make(map[string]func() status)
 	for _, src := range testPods {
 		for _, dst := range testPods {
+			if src == dst {
+				continue // minikube bug prevents this scenario from working
```

Comment:
May be qualify a bit more? Add a -minikube flag to bazel.

Reply:
Only the t->t scenario causes issues in minikube, I have this in my PR as well.